qutip/core/_brtensor.pyx

```diff
@@ -265,14 +265,14 @@ cdef class _BlochRedfieldElement(_BaseElement):
             return _br_term_data(A_eig, self.spectrum, self.skew, cutoff)
         raise ValueError('Invalid tensortype')
 
-    cpdef object qobj(self, double t):
```

Comment:
I did a few tests and it's actually faster in average:
For string and array based coefficient there is a small (~10 ns) cost.
But function coefficient are faster since they already return a python object and we did a `python -> C -> python` transformations.